.circleci/config.yml

```diff
@@ -28,12 +28,12 @@ jobs:
             cd ~/project/app/rust
             cargo clippy --version
             cargo clippy --all-features --all-targets || true
-      - run:
-          name: audit
-          command: |
-            cd ~/project/app/rust
-            cargo audit --version
-            cargo audit
+#      - run:
```

Comment:
I think we can reenable audit now.. the problems upstream have been fixed! 🎉

Reply:
Great! I'll re-enable it

Reply:
We still have the same problem with `cargo audit` https://app.circleci.com/pipelines/github/Zondax/ledger-polkadot/311/workflows/71363617-2cac-445d-82e3-ade8a7149a16/jobs/1418/parallel-runs/0/steps/0-104?invite=true

Reply:
yes, we need to upgrade dependencies too. 1 min, I commit that